build.sh

```diff
@@ -37,51 +37,58 @@ function getPlatformName() {
   fi
 }
 PLATFORM_NAME=$(getPlatformName)
-BUILD_TOOL="xbuild"
+BUILD_TOOL="msbuild"
 if isWindows; then
   BUILD_TOOL="MSBuild.exe"
 fi
 
-
 retrievePackage() {
   TYPE=$1
   PACKAGE=$2
+  TARGETDIR=$3
 
   pushd "${DOWNLOAD_DIR}"
-  if [ ! -f "${PACKAGE}.zip" ]; then
-    spatial package retrieve --force "${TYPE}" "${PACKAGE}" "${SDK_VERSION}" "${PACKAGE}.zip"
-    unzip -o "${PACKAGE}.zip"
+  if [ ! -d "${TARGETDIR}" ]; then
+    spatial package get --force --unzip "${TYPE}" "${PACKAGE}" "${SDK_VERSION}" "${TARGETDIR}"
   fi
   popd
 }
 
 # Get the tools:
 # * Spatial Schema compiler
 # * Standard Library Schemas
-# * Core SDK for all platforms to enable building workers for MacOS, Windows or Linux
 # * C# SDK
-retrievePackage "tools" "schema_compiler-x86_64-${PLATFORM_NAME}"
-retrievePackage "schema" "standard_library"
-retrievePackage "worker_sdk" "csharp"
-retrievePackage "worker_sdk" "core-dynamic-x86_64-win32"
-retrievePackage "worker_sdk" "core-dynamic-x86_64-linux"
-retrievePackage "worker_sdk" "core-dynamic-x86_64-macos"
+# * Core SDK for all platforms to enable building workers for MacOS, Windows or Linux
+retrievePackage "tools" "schema_compiler-x86_64-${PLATFORM_NAME}" "schema_compiler"
+retrievePackage "schema" "standard_library" "standard_library"
+retrievePackage "worker_sdk" "csharp" "lib/csharp"
+retrievePackage "worker_sdk" "core-dynamic-x86_64-win32" "lib/win64"
+retrievePackage "worker_sdk" "core-dynamic-x86_64-linux" "lib/linux64"
+retrievePackage "worker_sdk" "core-dynamic-x86_64-macos" "lib/macos64"
+
+# Generate C# code from the schemas:
+mkdir -p "${DOWNLOAD_DIR}"/csharp
```

Comment:
Nit: How about creating a OUT_DIR variable that we can reuse in line 74? That would make it more clear what we are doing here. Also, how about renaming the dir name slightly since (1) conceptually, it's not part of the our worker packages, how about putting the folder into something like`/SpatialOS/schema/generated/csharp`? While we are at it, how about factoring `${BUILD_DIR}"/SpatialOS/schema` into its own variable (something like `SCHEMA_DIR`)?